src/Makefile

```diff
@@ -45,7 +45,7 @@ SRCS=		rdkafka.c rdkafka_broker.c rdkafka_msg.c rdkafka_topic.c \
 		rdkafka_msgset_writer.c rdkafka_msgset_reader.c \
 		rdkafka_header.c rdkafka_admin.c rdkafka_aux.c \
 		rdkafka_background.c rdkafka_idempotence.c \
-		rdvarint.c rdbuf.c rdunittest.c \
+		rdvarint.c rdbuf.c rdunittest.c rdkafka_sasl_oauthbearer.c \
```

Comment:
See how rdkafka_sasl_scram.c is added to SRCS

Reply:
👍